openpdf/src/main/java/com/lowagie/text/pdf/hyphenation/HyphenationTree.java

```diff
@@ -350,7 +350,7 @@ public Hyphenation hyphenate(char[] w, int offset, int len,
         String sw = new String(word, 1, len);
         if (stoplist.containsKey(sw)) {
             // assume only simple hyphens (Hyphen.pre="-", Hyphen.post = Hyphen.no = null)
-            ArrayList hw = (ArrayList)stoplist.get(sw);
+            List hw = stoplist.get(sw);
```

Comment:
Please specify `List` items' type

Reply:
It would be nice but the `stoplist` property, according to the javadoc of the `HyphenationTree#addException()` on line 432, is alternatively filled with String and Object

```java
    /**
     * Add an exception to the tree. It is used by
     * {@link SimplePatternParser SimplePatternParser} class as callback to
     * store the hyphenation exceptions.
     * @param word normalized word
     * @param hyphenatedword a vector of alternating strings and
     * {@link Hyphen hyphen} objects.
     */
    public void addException(String word, ArrayList hyphenatedword) {
        stoplist.put(word, hyphenatedword);
    }

```

Reply:
That is ok, for me.